hw/dv/data/master_sim_cfgs_list.hjson

```diff
@@ -2,6 +2,10 @@
 // Licensed under the Apache License, Version 2.0, see LICENSE for details.
 // SPDX-License-Identifier: Apache-2.0
 {
+
```

Comment:
Please provide this a name as well:
```
  name: all_ip_lints // or similar
```

Reply:
looks like both the simulation master cfg and the lint master cfg are missing a name. how about
```
name: lint_all_batch
name: sim_all_batch
```
or
```
name: all_ip_lints
name: all_ip_sims
```

Reply:
See `hw/top_earlgrey/dv/top_earlgrey_sim_cfgs.hjson`
Admittedly, the name I am using there is not so great either. 
I wanted the name to reflect "All IPs. core and chip level DV sims for top_earlgrey" in a concise way.

Reply:
renamed them as 
```
top_earlgrey_batch_lint
top_earlgrey_batch_sim
```